contrib/scripts/avocado-fetch-eggs.py

```diff
@@ -51,6 +51,17 @@ def main():
         url = get_egg_url(python_version=version)
         asset = Asset(url, cache_dirs=CACHE_DIRS)
         asset.fetch()
+    # for version 3.12, it'll only be available after 103.0 is released
+    # TODO: remove after 103.0 is released
```

Comment:
@richtja https://github.com/avocado-framework/avocado/issues/5791 tracks this TODO.